packages/suite/src/components/firmware/FirmwareInitial.tsx

```diff
@@ -166,11 +167,7 @@ export const FirmwareInitial = ({
     const isBitcoinOnlyAvailable = !!device.firmwareReleaseConfigInfo?.isBitcoinOnlyAvailable;
 
     const installFirmware = (firmwareType: FirmwareType) => {
-        if (
-            isDeviceConnectedViaBluetooth &&
-            typeof device?.features.soc === 'number' &&
-            device.features.soc <= DEVICE_LOW_BATTERY_PERCENTAGE_THRESHOLD
-        ) {
+        if (isDeviceConnectedViaBluetooth && isDeviceLowOnBattery) {
```

Comment:
nit: would it make sense if the `isDeviceLowOnBattery` would return `false` in case that `isDeviceConnectedViaBluetooth` is equal to `false`? Then there would be no neet to always check both values and you could do
```suggestion
        if (isDeviceLowOnBattery) {
```

Reply:
I believe `isDeviceLowOnBattery` should only return if the device is low on battery no matter what transport it uses or atleast thats what the `isDeviceLowOnBattery` explicitly says.

I can imagine that, in the future, someone could be mistaken by this name and could be surprised that it also checks the connection transport. 

We can use only one selector, but it should be named something like `isBleDeviceLowOnBattery` IMHO or something similar.